src/vs/workbench/parts/execution/electron-browser/terminal.ts

```diff
@@ -11,6 +11,8 @@ if (env.isLinux) {
 		defaultTerminalLinux = 'x-terminal-emulator';
 	} else if (process.env.DESKTOP_SESSION === 'gnome' || process.env.DESKTOP_SESSION === 'gnome-classic') {
 		defaultTerminalLinux = 'gnome-terminal';
+	} else if (process.env.DESKTOP_SESSION === 'kde-plasma') {
+		defaultTerminalLinux = 'konsole';
```

Comment:
Is there some environment variable that houses this? `konsole` is the default terminal for this distribution, correct?